src/Dapr.Actors/Helper.cs

```diff
@@ -26,5 +27,19 @@ public static string GetCallContext()
                 return Guid.NewGuid().ToString();
             }
         }
+
+        public static string GetReentrancyContext()
```

Comment:
These methods should be defined on `ActorLogicalCallContext`. Putting them on a class called `Helper` just obfuscates their purpose.

Reply:
Gotcha. I was following the example set by the Helper class which appeared to be handling a similar context. From what you say below though, it sounds like this may not have been the best example to go off of.